kpatch-build/kpatch-build

```diff
@@ -1161,11 +1172,7 @@ else
 fi
 
 cd "$TEMPDIR/patch" || die
-if [[ -z "$OOT_MODULE" ]]; then
-	KPATCH_BUILD="$SRCDIR"
-else
-	KPATCH_BUILD="/lib/modules/$ARCHVERSION/build"
-fi
+KPATCH_BUILD="$SRCDIR"
```

Comment:
This line can be removed in favor of changing the below `export KPATCH_BUILD="$KPATCH_BUILD"` to `export KPATCH_BUILD="$SRCDIR"`.

Reply:
Will do